File: cpp/56d_1553A_DigitsSum.cpp
#include<bits/stdc++.h>

int solve(int n){
    return (n+1)/10;
}

int main(){
    int t, n;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>n;
        ans.push_back(solve(n));
    }
    for (auto i : ans){
        std::cout<<i<<'\n';
    }
    return 0;
}

File: python/56d_1553A_DigitsSum.py
def solve(n):
    return (n+1) // 10

def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        ans.append(solve(n))
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()